frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java

```diff
@@ -62,7 +68,13 @@ public final void actionPerformed(AnActionEvent e) {
 
   @UIEffect
   public boolean isSelected(AnActionEvent anActionEvent) {
-    return getGraphTable(anActionEvent).isListingCommits();
+    var presentation = anActionEvent.getPresentation();
+    return Toggleable.isSelected(presentation);
+  }
+
+  @UIEffect
+  public boolean isSelected(Presentation presentation) {
```

Comment:
I think this can be private

Reply:
Hmm seems it's still public (?)

Reply:
Now it's `private` (after force-push)  ;)

Reply:
Oh, I corrected first one and forgot correct the second. Done